<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportWidget</class>
 <widget class="QWidget" name="exportWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export Media</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="vl_1" stretch="1,8">
     <item>
      <layout class="QHBoxLayout" name="hb_1">
       <item>
        <widget class="QLabel" name="e_outPath_Label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="e_outPathEdit">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="e_selectOutBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select Destination</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hb_2" stretch="0,3">
       <item>
        <layout class="QGridLayout" name="control_layout" columnstretch="3,3,3">
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="vCodec_vLayout">
           <item>
            <widget class="QLabel" name="e_vCodecLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Video Codec:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="e_vCodecBox">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="format_vLayout" stretch="0,0">
           <item>
            <widget class="QLabel" name="e_formatLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Target Format:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="e_formatBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="2">
          <spacer name="space_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="e_startBtn">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="e_cancelBtn">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <layout class="QVBoxLayout" name="bitRate_vLayput">
           <item>
            <widget class="QLabel" name="e_bitRateText">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Bit Rate:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="mbps_hLayout">
             <item>
              <widget class="QComboBox" name="e_bitRateBox">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="e_mbpsText">
               <property name="font">
                <font>
                 <family>Bahnschrift Light</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Mbps</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <spacer name="space_1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1">
          <spacer name="space_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="e_opModeLabel">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Operation Mode:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="e_opModeBox">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Transcode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rewrap</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <layout class="QVBoxLayout" name="res_vLayout">
           <item>
            <widget class="QLabel" name="e_resLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Resolution:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="e_resBox">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QVBoxLayout" name="fps_vLayout">
           <item>
            <widget class="QLabel" name="e_fpsLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>FPS:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="e_fpsBox">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="2">
          <layout class="QVBoxLayout" name="aCodec_vLayout">
           <item>
            <widget class="QLabel" name="e_aCodecLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Audio Codec:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="e_aCodecBox">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="e_log_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
